tools/docker/mu-plugins/01-monorepo.php

```diff
@@ -39,7 +39,14 @@ class Monorepo {
 	 * Class constructor.
 	 */
 	public function __construct() {
-		$this->monorepo = '/usr/local/src/jetpack-monorepo/';
+		/**
+		 * Filter the monorepo path for development environments.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param string $path Monorepo file path.
+		 */
+		$this->monorepo = apply_filters( 'jetpack_monorepo_path', '/usr/local/src/jetpack-monorepo/' );
```

Comment:
I was dealing with the symlink issue on local environment without Docker. 

What about defining JETPACK_MONOREPO_PATH? So we just define it directly in the wp-config.php file.

`$this->monorepo = defined( 'JETPACK_MONOREPO_PATH' ) ? JETPACK_MONOREPO_PATH : '/usr/local/src/jetpack-monorepo/';`

Reply:
I have no preference between a filter vs a constant. I find I put a lot of local dev overrides in a `0-local.php` mu-plugin file, so a filter fits in in well here.

But a constant makes a lot of sense, as the monorepo path isn't something that's likely to change dynamically.

For reference, here's some of the content of my `0-local.php` mu-plugin file:
```
// Register theme project directories (that are symlinked to wp-content/).
register_theme_directory( 'wpcom-themes' );
register_theme_directory( 'theme-experiments' );

// Compensate for symlinked wpcomsh mu-plugin.
add_action( 'plugins_url', function( $url ) {
	if ( false !== strpos( $url, 'wpcomsh' ) ) {
		return str_replace( '/plugins/Users/Grant/Automattic/Sites', '/mu-plugins', $url );
	}

	return $url;
} );

// Run jetpack monorepo outside a docker container.
add_filter( 'jetpack_monorepo_path', function( $path ) {
	return '/Users/Grant/Automattic/Sites/jetpack/';
} );

// Fix an issue when running WordPress behind a reverse proxy,
// such as jurassic.tube tunneling for Jetpack development.
add_filter( 'redirect_canonical', function( $redirect_url, $requested_url ) {
	$redirect_host  = wp_parse_url( $redirect_url, PHP_URL_HOST );
	$requested_host = wp_parse_url( $requested_url, PHP_URL_HOST );

	if (
		// Is redirecting to the reverse proxy host
		$_SERVER['HTTP_X_FORWARDED_HOST'] === $redirect_host &&
		// and is requesting the local (valet) domain
		$_SERVER['HTTP_HOST'] === $requested_host
	) {
		return false;
	}

	return $redirect_url;
}, 10, 2 );
```